packages/card/src/react/index.js

```diff
@@ -61,7 +61,10 @@ const renderImage = props => (props.image ? props.image : null)
 
 const FullOverlay = glamorous.div(
   css['.psds-card__full-overlay'],
-  { ':focus-within': css['.psds-card__full-overlay:focus-within'] },
+  {
+    ':focus-within': css['.psds-card__full-overlay:focus-within'],
+    '[focus-within]': css['.psds-card__full-overlay:focus-within']
```

Comment:
I suppose the question will be: Why not add `[focus-within]` on the css side as well?  As I see it, [focus-within] is more a feature of the js implementation (that's where the polyfill is, after all).  If someone was to use the stylesheet, they'd need their own markup/code implementation.

Reply:
that makes sense to me